framework/test/unit/modules/interoperability/sidechain/commands/mainchain_registration.spec.ts

```diff
@@ -415,9 +409,69 @@ describe.skip('Mainchain registration command', () => {
 			// Assert
 			expect(ownChainAccountSubstore.set).toHaveBeenCalledWith(
 				expect.anything(),
-				getIDAsKeyForStore(0),
+				EMPTY_BYTES,
 				expectedValue,
 			);
 		});
+
```

Comment:
missing test for `EVENT_NAME_INVALID_REGISTRATION_SIGNATURE`?

Reply:
I think it is checked at L336

Reply:
I think in L336, you would need to check what event is called?

Reply:
To be addressed in: https://github.com/LiskHQ/lisk-sdk/issues/7546 once `eventQueue.add` is updated with `InvalidRegistrationSignatureEvent`